docs/settings.rst

```diff
@@ -58,6 +59,11 @@ ENCRYPT_BACKUP_CODES
 Backup codes to be encrypted before saving.
 Default: ``True``
 
+SECRET_KEY_LENGTH
+"""""""""""""""""
+Length of the shared secret key.  Secret key can be either 16 or 32 characters long.
```

Comment:
Let's move on with this feature.

I'd suggest we don't limit length, just inform user that iOs accepts 8, 16... and Android requires >= 16.